packages/web-components/src/components/ic-card/ic-card.stories.mdx

```diff
@@ -454,10 +456,12 @@ import Button from "../ic-button/readme.md";
           d="M17.65 6.35C16.2 4.9 14.21 4 12 4c-4.42 0-7.99 3.58-7.99 8s3.57 8 7.99 8c3.73 0 6.84-2.55 7.73-6h-2.08c-.82 2.33-3.04 4-5.65 4-3.31 0-6-2.69-6-6s2.69-6 6-6c1.66 0 3.14.69 4.22 1.78L13 11h7V4l-2.35 2.35z"
         />
       </svg>
-      <div slot="interaction-controls" style="display:flex;gap:16px;">
-        <ic-button variant="primary">Learn more</ic-button>
-        <ic-button variant="secondary">Hide</ic-button>
-      </div>
```

Comment:
I would imagine consumers have been using it with a 'interaction-controls' wrapper. How does it look with your CSS changes because when this update is released, it might unexpectedly change the styling on external apps.

Reply:
Were you able to test the previous implementation above but with the new styling?

Reply:
If an `interaction-controls` wrapper is used, then the user will mostly see the effects of the new CSS if they have more than one element with the slot attribute: `slot="interaction-controls"`. Below is how it looks if I were to use a wrapper. I believe the only difference a user using an `interaction-controls` wrapper would see is the positioning of the chevron. It will now appear at the bottom right of the card. 

The rest of the new CSS won't be applied inside the wrapper. As there's only one I wouldn't want to add styling a slotted div (`::slotted(div)`) as that could cause unintended side effects. 
![Screenshot 2023-04-18 at 14 20 54](https://user-images.githubusercontent.com/113986285/232790404-af455754-5204-454d-9432-544a123a5700.png)